src/ui/popup.ts

```
@@ -88,6 +88,11 @@ export type PopupOptions = {
      * @defaultValue false
      */
     subpixelPositioning?: boolean;
+    /**
+     * Optional opacity when the location is behind the globe.
+     * If not supplied default to 1. Note the number is converted to a string.
```

Why not allow defining a string then?

---

Also use the `@defaultValue` like other options.

---

Since we are defining `@defaultValue`, should we make it 0 to hide the popup when behind by default? That seems like it should be expected behaviour.

---

Wouldn't this change the current behavior for people not setting this value and create breaking change?

---

Yes that's why I'm asking. But this issue was created in the expectation that the opacity should be lower behind the globe. Is globe a fairly recent change? 

Also I had a colleague point out that they would expect it to be zero behind by default. 

We can default it to 1 and get rid of an if statement also.

---

Default should keep the current behavior, we can consider changing that in next major version release.

---

Yes, globe was introduced early this year.